src/qutip_cupy/dense.py

```diff
@@ -116,17 +321,82 @@ def cupydense_from_dense(dense):
     return dense_cp
 
 
-def cpd_adjoint(cpd_array):
+def adjoint_cupydense(cpd_array):  # noqa: E302
```

Comment:
E302 is "Expected two blank lines". Why do we ignore that here? It looks like there are two blank lines above this function definition? Same question for other functions.

Reply:
I might have had a mix-up here, but I thought that we inclined to having smaller spacing of short funtions, so that the files do not become to long and is eadier for reading.

Reply:
I would say either commit to the code style, or disable that warning globally.  Personally I don't see any problem with the extra spacing - it's better to be consistent, and have a tool you can use to say "this is always the right way to do it" than to say "this is mostly how we do it, but then there's a bunch of exceptions".  In the latter case, you just end up with a higgledy piggledy mess, and you spend a bunch of time arguing in code review with contributors that "this patch is formatted really badly, please fix", and they say "but _your_ code has all sorts of exceptions".  Better to just be hard-nosed about it and say, "it might not be universally perfect, but consistent and unalienable is better in the long run".